src/lib/y2storage/proposal/autoinst_drives_map.rb

```diff
@@ -97,6 +97,13 @@ def partitions?
         @drives.values.any? { |i| !i.partitions.empty? }
       end
 
+      # Determine whether any of the drives sets snapshots enabled
+      #
+      # @return [Boolean]
+      def use_snapshots?
+        @drives.empty? || @drives.values.any?(&:enable_snapshots)
```

Comment:
I believe that, to be consistent with the current philosophy, the decision about what to do if the field is nil in the profile should be taken here.

Reply:
The `enable_snapshots` field, I meant.